tests/test_util_modules.py

```diff
@@ -131,6 +131,8 @@ def test_vector_compression(size=(128, 128, 64), alpha=5e-08):
     assert error.square().mean() < alpha
     error = deserialize_torch_tensor(serialize_torch_tensor(X, CompressionType.FLOAT16)) - X
     assert error.square().mean() < alpha
+    error = deserialize_torch_tensor(serialize_torch_tensor(X, CompressionType.QUANTILE_8BIT)) - X
+    assert error.square().mean() < 0.0008
```

Comment:
We'd better parametrize this tolerance by a constant (by the way, why not torch.allclose here?)

Reply:
@allclose @mryab 
Thing is, allclose measures a significantly different statistic. In this test, we check for mean squared deviation, while allclose would react to the __worst__ case, even if it is just one number in a million.